ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -318,6 +318,21 @@ const tableColumnDescriptor: Readonly<IntegrationTableColumnDescriptorMap> = {
                 Header: 'Schedule',
             },
         ],
+        s3compatible: [
+            { accessor: 'name', Header: 'Name' },
+            { accessor: 's3compatible.bucket', Header: 'Bucket' },
+            {
+                accessor: ({ schedule }) => {
+                    if (schedule.intervalType === 'WEEKLY') {
+                        return `Weekly on ${daysOfWeek[schedule.weekly.day]} @ ${
+                            timesOfDay[schedule.hour]
+                        } UTC`;
+                    }
+                    return `Daily @ ${timesOfDay[schedule.hour]} UTC`;
```

Comment:
Ouch, although better to be consistent, future design debt for UI team to confirm with tech docs that **at** is better than at symbol.